### ISSUE TITLE
Error moving completed files

### ISSUE THREAD

**Issue by joliett1234** [BiglySoftware/BiglyBT#1543]
I am getting errors when Bigly automatically moves completed downloaded files.  I am using valid directories but yet these errors appear: 
**Error moving completed files. Move of the file j:\Downloads\downloadFile.  Failed to create parent directory J:Videos\downloadFile_**

J:\Downloads\BigltBT Downloads <--- Default Directory works OK
J:\VIDEOS   <--- The directory I designate to  move completed files
j:\Download\BigltBT\Torrents <-- This directory I want to save torrent files does not work, torrent will not start.

I believe the problem stems from the fact that the Directories I am using are the system folders DOWNLOAD and VIDEOS that I moved years ago with Windows system from the standard location (c:\users\name\Download and c:\users\name\Videos) to a different local drive j:\Download.  I moved those folders to minimize the size of C: during system backups.

I see that on two occasions, Biggly created the directories BiglyBT and BiglyBT Downloads where it thought the VIDEOS and DOWNLOAD folder should be located in c:\users\username\Downloads and c:\users\username\videos

Can I use the directories I want to use?  j:\downloads and j:\Videos???
Thank you!

**Comment by joliett1234** [comment]
Now I just got this error at file completion:
Error moving completed files
Move of file J:\Downloads\BigltBT Downloads\File [1080p]\File...failed, Failed to create parent directory 'J:\Videos\File (1973) [1080p]'..

**Comment by parg** [comment]
Is it a file permissions problem? Have you tried manuallyh creating the directory that BiglyBT is failing on?

**Comment by joliett1234** [comment]
I read it may be a file permission problem, but had no idea how to do it.  Seeing your response gave me impetus to delve in.

I wasn't really sure how to check the file permission....but I am an advanced computer user...so I went into the directory I am using j:\Videos.  I right clicked j:\Videos, security, and I wasn't sure which Group or username to use, so I went through and gave full permission to every Group or username. **It worked**!

Thank you parg, I'm loving Bigly.
